heron/tracker/src/python/metricstimeline.py

```diff
@@ -112,7 +110,7 @@ def getMetricsTimeline(tmaster,
 
   if metricResponse.status.status == common_pb2.NOTOK:
     if metricResponse.status.HasField("message"):
-      raise Exception(metricResponse.status.message)
+      Log.error(metricResponse.status.message)
```

Comment:
What is the difference here? In case raise Exception and terminate - where as in the Log.error case, it will print the error and continue with the execution?